### ISSUE TITLE
Re-streams and Proof of Popularity

### ISSUE THREAD

**Issue by Matlemad** [livepeer/livepeer-monorepo#102]
**What's the problem? (required)**
I'm limited to the Livestreams I produce, I can't re-stream something using the LP protocol. 

**What's the current behavior? (required)**
 the only way I can share a relevant livestream right now it is by using the old system of copying the url or embedding the video on a page. That would result in the transcoder having to keep up to the rising demand (right?).

**Describe a possible solution you've considered (optional)**
LP protocol could feature a RE-STREAM function, where I take the "Ox..." of a given livestream and 
paste it somewhere in Terminal. By doing so I basically help that livestream being spread by paying myself the job to my transcoder.

**Proof of Popularity**
it's a reputational system that rewards people re-streaming relevant livestreams, and by doing so 1) defines communities using livepeer 2) monitors the quality of the content being live-streamed.
It could result in transcoders working with each other and sharing rewards for a particular livestream and users being able to consolidate live-streaming relationships around the world.
I still have not clear how that could work in terms of ETH or LPT. It's just an idea that came up to me (named "proof of popularity" by my pal Redvers Parson) and I thought it was worth-sharing :)

**Comment by jozanza** [comment]
Thanks for the POP description! It sounds intriguing, although I'm not so sure how to approach anything actionable with it. Maybe you can even write up a white paper on it 💯?

But re-streaming, on the other hand, is definitely solvable. However, it's not something that can be addressed with the livepeerjs codebase. You may want to check out the [existing issues on this topic in the go-livepeer repo](https://github.com/livepeer/go-livepeer/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+restream).

**tl;dr;** It seems like there are already some existing services you could take advantage of. There's one called [Restream](https://restream.io/) that looks pretty cool. Maybe there are also some OBS plugins I don't know about.

Good luck @Matlemad !
